server/controllers/refer/referralsController.test.ts

```diff
@@ -265,9 +266,6 @@ describe('ReferralsController', () => {
 
   describe('checkAnswers', () => {
     it('renders the referral check answers page', async () => {
-      const errors: Array<string> = []
-      request.flash = jest.fn().mockReturnValue(errors)
```

Comment:
Are we happy with the call to `flash` being unmocked?

Reply:
This was meant in reference to the final commit - does the flash not need mocking there? Perhaps not

Reply:
(I suppose it makes no difference to the outcome of the action in those conditions, but it's still hit)

Reply:
`.flash` should already be mocked as a `jest.fn()` as a result of `request = createMock<Request>({ user: { token } })` so it doesn't need it here as well.